src/components/ToolTip.tsx

```diff
@@ -102,15 +104,67 @@ export default function ({
           </div>
         </div>
       )}
-      <div
-        ref={childrenRef}
-        className={tooltipChildrenWrapper}
-        onMouseEnter={() => setIsShow(true)}
-        onMouseLeave={() => setIsShow(false)}
-        onClick={() => setIsShow(true)}
-      >
-        {children}
-      </div>
+      {position === 'floating' ? (
+        <div
+          ref={childrenRef}
+          className={tooltipChildrenWrapper}
+          onMouseMove={(e) => {
```

Comment:
It's better to extract this into function

Reply:
Did it, thx